src/lib/components/auth-cards/LaunchCard.svelte

```diff
@@ -18,11 +17,14 @@
   export let height;
   export let store;
   export let custom;
+  export let test = false;
```

Comment:
We can probably rename this to something more intuitive. I know this is existing code but still..

Reply:
Yes, this is for storybook testing. Agreed.